views/note.ejs

```diff
@@ -0,0 +1,26 @@
+<%- include("partials/head.ejs") %> <%- include("partials/header.ejs") %>
+
+<h1><%= job ? "Edit Note Entry" : "Add New Note Entry" %></h1>
+<form method="POST" action="<%= note ? '/notes/update/' + note.id : '/notes' %>" >
+  <input type="hidden" name="_csrf" value="<%= _csrf %>" />
+  <div>
+    <label for="title">Title:</label>
+    <input name="title" id="title" value="<%= note ? note.title : '' %>" />
+  </div>
+  <div>
+    <label for="subject">Subject:</label>
+    <input name="subject" id="subject" value="<%= note ? note.position : '' %>"/>
```

Comment:
Change note.position for note.subect here.

Reply:
Done